### PR TITLE
fixes the pump ui clicktips

### PR DESCRIPTION
**Pull request by factorsofx**
## About the PR
This PR fixes the previously merged but perpetually broken patch I made in 2018 that lets you click on a pump and adjust it. 

## Why's this needed?
This would allow marginally more tinkering with engine/toxins stuff and is a first step towards some more customizable atmospheric machinery stuff I want to make eventually.

### CODE REVIEW

File: code/modules/atmospherics/machinery/binary/passive_gate.dm [hunk 1]
```diff
@@ -59,10 +59,9 @@ obj/machinery/atmospherics/binary/passive_gate
 			if(network2)
 				network2.update = 1
 
-obj/machinery/atmospherics/binary/passive_gate/attackby(obj/weapon as obj, mob/user as mob)
+obj/machinery/atmospherics/binary/passive_gate/attack_hand(mob/user as mob)
 	..()
-	if(istype(weapon, /obj/item/device/multitool))
```
Comment (ZeWaka) [inline code/modules/atmospherics/machinery/binary/passive_gate.dm:64]: Why remove the multitool check?

Comment (factorsofx) [reply code/modules/atmospherics/machinery/binary/passive_gate.dm:64]: oh I forgot I changed it to test it easier
will add that back in a jiffy

**Comment by factorsofx** [comment]
Is fixed @ZeWaka

File: code/modules/atmospherics/machinery/binary/passive_gate.dm [hunk 2]
```diff
@@ -59,9 +59,8 @@ obj/machinery/atmospherics/binary/passive_gate
 			if(network2)
 				network2.update = 1
 
-obj/machinery/atmospherics/binary/passive_gate/attackby(obj/weapon as obj, mob/user as mob)
-	..()
```
Comment (ZeWaka) [inline code/modules/atmospherics/machinery/binary/passive_gate.dm:63]: Why get rid of the parent call?

Comment (factorsofx) [reply code/modules/atmospherics/machinery/binary/passive_gate.dm:63]: that way it doesn't show the "elan oots hits the ___ pump with a multitool!"

File: code/modules/atmospherics/machinery/binary/passive_gate.dm [hunk 3]
```diff
@@ -59,9 +59,8 @@ obj/machinery/atmospherics/binary/passive_gate
...
-	if(istype(weapon, /obj/item/device/multitool))
+obj/machinery/atmospherics/binary/passive_gate/attackby(obj/item/W as obj, mob/user as mob)
```
Comment (Rohesie) [inline code/modules/atmospherics/machinery/binary/passive_gate.dm:62]: `as obj` or `as mob` do nothing if this proc is not used as a verb. And one would hope it isn't.

File: code/modules/atmospherics/machinery/binary/passive_gate.dm (hunk 2 above)
Comment (factorsofx) [reply code/modules/atmospherics/machinery/binary/passive_gate.dm:63]: wait I should do the parent call in the else

File: code/modules/atmospherics/machinery/binary/passive_gate.dm (hunk 3 above)
Comment (ZeWaka) [reply code/modules/atmospherics/machinery/binary/passive_gate.dm:62]: I'll fix that.

**Pull request merged**